icons/cloudflare/cloudflare-original-wordmark.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#F6821F" d="m107.119 69.493.303-1.048c.361-1.246.227-2.398-.379-3.245-.558-.78-1.487-1.239-2.615-1.292l-21.37-.27a.422.422 0 0 1-.337-.179.43.43 0 0 1-.048-.385.574.574 0 0 1 .496-.379l21.568-.273c2.558-.117 5.328-2.192 6.298-4.721l1.23-3.211a.78.78 0 0 0 .033-.425c-1.397-6.277-7.003-10.971-13.706-10.971-6.176 0-11.42 3.983-13.301 9.52a6.306 6.306 0 0 0-4.437-1.227c-2.963.294-5.345 2.678-5.639 5.639a6.36 6.36 0 0 0 .163 2.207c-4.84.141-8.722 4.104-8.722 8.975.001.435.033.869.096 1.299a.416.416 0 0 0 .412.359l39.452.005h.011a.519.519 0 0 0 .492-.378z"/><path fill="#FBAD41" d="M114.238 54.722c-.198 0-.395.005-.592.015a.345.345 0 0 0-.308.247l-.84 2.9c-.361 1.246-.227 2.397.379 3.244.557.781 1.486 1.239 2.614 1.292l4.556.273a.404.404 0 0 1 .324.176.426.426 0 0 1 .048.387.57.57 0 0 1-.495.378l-4.733.273c-2.57.118-5.339 2.192-6.308 4.721l-.342.893a.245.245 0 0 0-.015.117.257.257 0 0 0 .24.225h16.3a.43.43 0 0 0 .42-.313c.289-1.029.434-2.092.433-3.161-.001-6.442-5.23-11.667-11.681-11.667z"/><path fill="#222" d="M15.792 74.688h2.779v7.585h4.856v2.433h-7.634V74.688zM26.304 79.726v-.028c0-2.877 2.32-5.21 5.414-5.21s5.385 2.305 5.385 5.181v.029c0 2.877-2.321 5.208-5.413 5.208s-5.386-2.304-5.386-5.18zm7.963 0v-.028c0-1.444-1.045-2.705-2.577-2.705-1.518 0-2.535 1.231-2.535 2.677v.029c0 1.444 1.046 2.705 2.564 2.705 1.531-.002 2.548-1.232 2.548-2.678zM40.503 80.313v-5.625h2.822v5.568c0 1.444.731 2.132 1.847 2.132 1.117 0 1.847-.658 1.847-2.061v-5.639h2.822v5.552c0 3.235-1.847 4.651-4.698 4.651s-4.64-1.444-4.64-4.579M54.094 74.689h3.867c3.581 0 5.657 2.061 5.657 4.951v.029c0 2.89-2.105 5.037-5.714 5.037h-3.81V74.689zm3.91 7.555c1.662 0 2.764-.914 2.764-2.534v-.028c0-1.602-1.102-2.533-2.764-2.533h-1.132v5.096l1.132-.001zM67.656 74.688h8.02v2.433h-5.241v1.702h4.74v2.304h-4.74v3.578h-2.779V74.688zM79.543 74.688h2.778v7.585h4.856v2.433h-7.634V74.688zM94.437 74.617h2.677l4.269 10.089h-2.979l-.731-1.789h-3.867l-.716 1.789h-2.922l4.269-10.089zm2.435 6.139-1.117-2.848-1.132 2.848h2.249zM104.962 74.688h4.74c1.533 0 2.592.401 3.265 1.088.588.573.888 1.347.888 2.333v.028c0 1.531-.819 2.547-2.062 3.076l2.392 3.493h-3.209l-2.019-3.035h-1.217v3.035h-2.778V74.688zm4.612 4.809c.945 0 1.49-.458 1.49-1.188v-.029c0-.787-.573-1.187-1.505-1.187h-1.818v2.406l1.833-.002zM117.865 74.688h8.063v2.362h-5.313v1.516h4.812v2.19h-4.812v1.589H126v2.361h-8.135V74.688zM9.709 80.9c-.389.879-1.208 1.502-2.296 1.502-1.519 0-2.564-1.259-2.564-2.705v-.029c0-1.444 1.017-2.677 2.535-2.677 1.145 0 2.016.704 2.384 1.661h2.929c-.469-2.384-2.566-4.164-5.284-4.164C4.319 74.488 2 76.823 2 79.697v.028c0 2.877 2.291 5.181 5.384 5.181 2.646 0 4.714-1.712 5.26-4.007l-2.935.001z"/></svg>
```

Comment:
Not sure where this logo is from, but the official one has a different color palette. The official logo has this style
```svg
<style type="text/css">
	.st0{fill:#FFFFFF;}
	.st1{fill:#F38020;}
	.st2{fill:#FAAE40;}
</style>
```
You can download all their logos here -> https://www.cloudflare.com/logo/
![cf-logo-v-rev](https://user-images.githubusercontent.com/43886029/180870092-43c42129-0b32-4bf8-ab19-b3be84082108.svg)

Reply:
I used like indicated in the PR the logo SVG of there website https://www.cloudflare.com/
the logo is located on the top left corner, it's a SVG, I used the colors values of this SVG.

Reply:
And I didn't used the logo of there brand kit since it's not an icon but a true logo, there is the littre ® for Registered on the right side. I didn't wanted to alter the logo. So I directly use the Website icon / logo where there isn't the ®.

Since the final product is an glyph in the devicon font, the ® would be too small to be correctly rendered.

Reply:
Those are some good reasons why you did like you did.

I still feel however that we should use the logo from their official logo kit, even if we have to alter it a little.

Here's the logo-kit version without the ® character.
![cf-logo-v-rev](https://user-images.githubusercontent.com/43886029/180875688-ae8fa9ab-e422-4572-ac2e-d90b1d04ddc4.svg)

```svg
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 874.3 393.7" style="enable-background:new 0 0 874.3 393.7;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#FFFFFF;}
	.st1{fill:#F38020;}
	.st2{fill:#FAAE40;}
</style>
<g>
	<polygon class="st0" points="155.1,259.1 170.7,259.1 170.7,301.6 197.8,301.6 197.8,315.2 155.1,315.2 	"/>
	<path class="st0" d="M213.9,287.3v-0.2c0-16.1,13-29.2,30.3-29.2s30.1,12.9,30.1,29v0.2c0,16.1-13,29.2-30.3,29.2
		C226.7,316.3,213.9,303.4,213.9,287.3 M258.5,287.3v-0.2c0-8.1-5.8-15.1-14.4-15.1c-8.5,0-14.2,6.9-14.2,15v0.2
		c0,8.1,5.8,15.1,14.3,15.1C252.8,302.3,258.5,295.4,258.5,287.3"/>
	<path class="st0" d="M293.4,290.6v-31.5h15.8v31.2c0,8.1,4.1,11.9,10.3,11.9s10.3-3.7,10.3-11.5v-31.6h15.8v31.1
		c0,18.1-10.3,26-26.3,26C303.4,316.2,293.4,308.2,293.4,290.6"/>
	<path class="st0" d="M369.4,259.1H391c20,0,31.7,11.5,31.7,27.7v0.2c0,16.2-11.8,28.2-32,28.2h-21.3L369.4,259.1L369.4,259.1z
		 M391.3,301.4c9.3,0,15.5-5.1,15.5-14.2v-0.2c0-9-6.2-14.2-15.5-14.2H385v28.5H391.3z"/>
	<polygon class="st0" points="445.3,259.1 490.2,259.1 490.2,272.7 460.8,272.7 460.8,282.3 487.4,282.3 487.4,295.2 460.8,295.2 
		460.8,315.2 445.3,315.2 	"/>
	<polygon class="st0" points="511.8,259.1 527.3,259.1 527.3,301.6 554.5,301.6 554.5,315.2 511.8,315.2 	"/>
	<path class="st0" d="M595.1,258.7h15l23.9,56.5h-16.7l-4.1-10h-21.6l-4,10h-16.3L595.1,258.7z M608.8,293.1l-6.2-15.9l-6.3,15.9
		H608.8z"/>
	<path class="st0" d="M654,259.1h26.5c8.6,0,14.5,2.2,18.3,6.1c3.3,3.2,5,7.5,5,13.1v0.2c0,8.6-4.6,14.3-11.5,17.2l13.4,19.6h-18
		l-11.3-17h-6.8v17H654L654,259.1L654,259.1z M679.8,286c5.3,0,8.3-2.6,8.3-6.6v-0.2c0-4.4-3.2-6.6-8.4-6.6h-10.2V286L679.8,286
		L679.8,286z"/>
	<polygon class="st0" points="726.2,259.1 771.3,259.1 771.3,272.3 741.6,272.3 741.6,280.8 768.5,280.8 768.5,293.1 741.6,293.1 
		741.6,302 771.7,302 771.7,315.2 726.2,315.2 	"/>
	<path class="st0" d="M121.1,293.9c-2.2,4.9-6.8,8.4-12.8,8.4c-8.5,0-14.3-7.1-14.3-15.1V287c0-8.1,5.7-15,14.2-15
		c6.4,0,11.3,3.9,13.3,9.3h16.4c-2.6-13.4-14.4-23.3-29.6-23.3c-17.3,0-30.3,13.1-30.3,29.2v0.2c0,16.1,12.8,29,30.1,29
		c14.8,0,26.4-9.6,29.4-22.4L121.1,293.9L121.1,293.9z"/>
	<g>
		<polygon class="st0" points="752.9,169.6 710,145 702.6,141.8 527.1,143 527.1,232.1 752.9,232.1 		"/>
		<path class="st1" d="M674.8,223.9c2.1-7.2,1.3-13.8-2.2-18.7c-3.2-4.5-8.6-7.1-15.1-7.4l-123.1-1.6c-0.8,0-1.5-0.4-1.9-1
			c-0.4-0.6-0.5-1.4-0.3-2.2c0.4-1.2,1.6-2.1,2.9-2.2l124.2-1.6c14.7-0.7,30.7-12.6,36.3-27.2l7.1-18.5c0.3-0.8,0.4-1.6,0.2-2.4
			c-8-36.2-40.3-63.2-78.9-63.2c-35.6,0-65.8,23-76.6,54.9c-7-5.2-15.9-8-25.5-7.1c-17.1,1.7-30.8,15.4-32.5,32.5
			c-0.4,4.4-0.1,8.7,0.9,12.7c-27.9,0.8-50.2,23.6-50.2,51.7c0,2.5,0.2,5,0.5,7.5c0.2,1.2,1.2,2.1,2.4,2.1l227.2,0
			c1.3,0,2.5-0.9,2.9-2.2L674.8,223.9z"/>
		<path class="st2" d="M714,144.8c-1.1,0-2.3,0-3.4,0.1c-0.8,0-1.5,0.6-1.8,1.4l-4.8,16.7c-2.1,7.2-1.3,13.8,2.2,18.7
			c3.2,4.5,8.6,7.1,15.1,7.4l26.2,1.6c0.8,0,1.5,0.4,1.9,1c0.4,0.6,0.5,1.5,0.3,2.2c-0.4,1.2-1.6,2.1-2.9,2.2l-27.3,1.6
			c-14.8,0.7-30.7,12.6-36.3,27.2l-2,5.1c-0.4,1,0.3,2,1.4,2h93.8c1.1,0,2.1-0.7,2.4-1.8c1.6-5.8,2.5-11.9,2.5-18.2
			C781.3,175,751.1,144.8,714,144.8"/>
	</g>
</g>
</svg>
```